lib/internal/modules/esm/resolve.js

```diff
@@ -310,7 +310,8 @@ function finalizeResolution(resolved, base, preserveSymlinks) {
       }
     }
 
-    path = file;
+    resolved = file;
+    path = fileURLToPath(resolved);
```

Comment:
This is outside of the `if (getOptionValue('--experimental-specifier-resolution') === 'node')` block -- does this mean it changes behaviour when `'--experimental-specifier-resolution` is _not_ specified?

Reply:
Oh that was a stupid mistake. Interestingly the tests did all pass - but I will change this. 

It's odd to me that without my change `path` changes from being a string to a URL

Reply:
hang on - it *is* in the if statement

Reply:
🤦 You're right. My mistake. Sorry.